src/controllers/controllerpresetinfoenumerator.cpp

```diff
@@ -29,6 +29,10 @@ bool presetInfoNameComparator(const PresetInfo &a, const PresetInfo &b) {
 }
 }
 
+PresetInfoEnumerator::PresetInfoEnumerator(const QString& searchPath)
+        : PresetInfoEnumerator(QList<QString>{searchPath}) {
+}
+
 PresetInfoEnumerator::PresetInfoEnumerator(const QStringList& searchPaths)
```

Comment:
remove this old constructor

Reply:
I thought I leave it in so that we can easily add additional directories later, for example to add support for the XDG Directory Spec while maintaining backward compatibility with mappings in `~/.mixxx`. I you insist, I can also remove that contructor.

Reply:
Okay, I didn't realize you had future plans for it.

Reply:
Yes, but not for 2.3 :-)